flow/envs/base_env.py

```diff
@@ -550,6 +480,11 @@ def reset(self):
         # update the colors of vehicles
         self.update_vehicle_colors()
 
+        # check to make sure all vehicles have been spawned
+        if len(self.initial_ids) < self.vehicles.num_vehicles:
+            logging.error("Not enough vehicles have spawned! Bad start?")
+            sys.exit()
```

Comment:
For debugging purpose, perhaps print which vehicles were not spawned and why?

Reply:
added information on the names of vehicles and their initial states for ease of debugging